db/toblock.c

```diff
@@ -2418,14 +2420,15 @@ static void backout_and_abort_tranddl(struct ireq *iq, tran_type *parent,
 {
     int rc = 0;
     if (rowlocks) {
+        tran_type *new_parent = bdb_get_sc_parent_tran(parent); // TODO: Verify this is correct.
         assert(parent);
```

Comment:
This is not necessary- the parent won't be touched - but (based on our conversation during the review) you could NULL the parent's physical transaction after this abort to guarantee a segfault should any later code attempt to access it.